### PR TITLE
Problem: `release/v4-croeseid` branch is not up-to-date

### PR DESCRIPTION
**Pull request by devashishdxt**
Solution: Merge `master` into `release/v4-croeseid`.

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+          src = libwasmvm-src + "/libwasmvm";
+          cargoDepsName = "vendor"; # use a static name to avoid rebuild when name changed
+          cargoSha256 = "sha256-m3CtXHAkjNR7t7zie9FWK4k5xvr6/O2BfGQYi+foxCc=";
+          doCheck = false;
+        };
+      } // (with final;
+        let
+          matrix = lib.cartesianProductOfSets {
+            db_backend = [ "goleveldb" "rocksdb" ];
+            network = [ "mainnet" "testnet" ];
```
Comment (tomtau) [inline flake.nix:96]: will the current build script create all those combinations of binaries? for release/v4-croeseid releases, only the testnet goleveldb binaries should be built?

**Review by tomtau — APPROVED** [review]

File: flake.nix (hunk 1 above)
Comment (devashishdxt) [reply flake.nix:96]: It'll create mainnet binaries. I'll create another PR to change that. This PR only merges `master` into the `release/v4-croeseid` branch.

**Comment by devashishdxt** [comment]
> /runsim

Seems that simulation tests are not triggered.

**Comment by allthatjazzleo** [comment]
> > /runsim
> 
> Seems that simulation tests are not triggered.
@devashishdxt @tomtau 
coz currently permission of ${{ secrets.GITHUB_TOKEN }} is read only
It requires to write to comment the PR

**Comment by tomtau** [comment]
maybe it needs to be merged first for it to be triggered?

**Comment by devashishdxt** [comment]
Should we create a cron job to run simulation tests once every day and create issue if it fails?

**Comment by tomtau** [comment]
will you rebase the commits onto [release/v4-croeseid](https://github.com/crypto-org-chain/chain-main/tree/release/v4-croeseid), so that there aren't those merge commits?
generally this porting to branches should be done via cherry-pick

**Comment by devashishdxt** [comment]
Ok. Will open a new PR.

**Pull request closed without merging**